pkg/console/status/status.go

```diff
@@ -64,6 +64,36 @@ func HandleUpgradable(typePrefix string, reason string, err error) v1helpers.Upd
 	return v1helpers.UpdateConditionFn(condition)
 }
 
+func (c *StatusHandler) ResetConditions(conditions []operatorsv1.OperatorCondition) []v1helpers.UpdateStatusFunc {
```

Comment:
I see your point, but I would rather have a generic function like this, that would loop though all the conditions in the main controller and reset them.
The point is that is more reliable then if we forget to add a new prefix to the controller's list of responsible conditions or even if a typo appears. 
Also once the `Removed` state is set on the operator we are not setting/updating any conditions, just logging into the operator's pod

Reply:
Thats why I would reset all the conditions in this main sync loop or create a something like a maintenance controller. But since we dont have any other use-cases for it I would just do the simple thing and reset it in the main controller.

Reply:
Sounds good to me!